### PR TITLE
Pass down chrony configuration to subprojects

### PR DESCRIPTION
**Pull request by zmc**

### CODE REVIEW

File: files/site.tfvars.in [hunk 1]
```diff
@@ -24,7 +24,7 @@ dns_forwarders              = "192.168.88.1;$DNS_BACKUP_SERVER"
 chrony_config               = true
 chrony_config_servers       = [$CHRONY_CONFIG_SERVERS]
 
-install_playbook_tag        = "9a5128fc58ed86aa332128314181dc9ab86334c6"
+install_playbook_tag        = "master"
```
Comment (lukebrowning) [inline files/site.tfvars.in:27]: I would feel better setting this to a specific commit if something is missing or broken.   If a bug is introduced here, we may lose a day or two as we are tracking other problems.   We are about to have ocs-ci running in automated fashion every day on a couple of servers.   That is going to take extra time and resources to keep track of.

File: files/site.tfvars.in [hunk 2]
```diff
@@ -21,6 +21,8 @@ cluster_domain              = "$CLUSTER_DOMAIN"
 cluster_id                  = "ocp$SANITIZED_OCP_VERSION"
 
 dns_forwarders              = "192.168.88.1;$DNS_BACKUP_SERVER"
+chrony_config               = true
```
Comment (lukebrowning) [inline files/site.tfvars.in:24]: I believe you also need to include the fix from ocp4-upi-kvm submodule.

File: files/site.tfvars.in (hunk 1 above)
Comment (zmc) [reply files/site.tfvars.in]: Okay, I've pointed it to a specific commit now.

File: files/site.tfvars.in (hunk 2 above)
Comment (zmc) [reply files/site.tfvars.in:24]: Good point; done.

**Pull request merged**